electron/main/index.ts

```diff
@@ -235,6 +235,20 @@ async function createWindow() {
     return await baseCoin.recover(parameters);
   });
 
+  ipcMain.handle('recoverConsolidations', async (event, coin, params) => {
+    switch (coin) {
+      case 'trx':
+      case 'ttrx': {
+        // Batch Consolidations only implemented for Tron for now so have
+        // to reference Trx Coin directly instead of Basecoin;
+        const trxCoin = sdk.coin(coin) as Trx | Ttrx;
+        return await trxCoin.recoverConsolidations(params);
+      }
+      default:
+        return new Error(`Coin: ${coin} does not support consolidation recovery`);
+    }
```

Comment:
can you wrap this in a try/catch and return the error to the UI thread?

Reply:
Pretty sure errors in the icp handlers are swallowed by the nodejs process and doesnt surface to the UI